elements/howto-menu/howto-menu.e2etest.js

```diff
@@ -0,0 +1,138 @@
+const helper = require('../../tools/selenium-helper.js');
+const expect = require('chai').expect;
+const {Key} = require('selenium-webdriver');
+
+describe('howto-menu', function() {
+  let success;
+  beforeEach(function() {
+    return this.driver.get(`${this.address}/howto-menu/demo.html`)
+      .then(_ => helper.waitForElement(this.driver, 'howto-menu'));
+  });
+
+  it('should focus the first item when opened',
+    async function() {
+      await this.driver.executeScript(_ => {
+        window.menu = document.querySelector('howto-menu');
+        window.menu.removeAttribute('hidden');
+        window.expectedMenuItem = document.querySelector('[role="menuitem"]');
+      });
+      success = await this.driver.executeScript(
+        _ => document.activeElement === window.expectedMenuItem
+      );
+      expect(success).to.equal(true);
+    }
+  );
+
+  it('should focus the next item on [arrow down]',
+    async function() {
+      await this.driver.executeScript(_ => {
+        window.menu = document.querySelector('howto-menu');
+        window.menu.removeAttribute('hidden');
+      });
+      await this.driver.actions().sendKeys(Key.ARROW_DOWN).perform();
+      success = await this.driver.executeScript(`
+        return document.activeElement ===
+          document.querySelectorAll('[role="menuitem"]')[1];
+      `);
+      expect(success).to.equal(true);
+    }
+  );
+
+  it('should wrap the focus at the end of menu on [arrow down]',
+    async function() {
+      await this.driver.executeScript(_ => {
+        window.menu = document.querySelector('howto-menu');
+        window.menu.removeAttribute('hidden');
+        const items = document.querySelectorAll('[role="menuitem"]');
+        items[items.length - 1].focus();
```

Comment:
I think this test works for this implementation but wouldn't work if we were using `aria-activedescendant` instead of roving tabindex. I'd suggest counting the number of menu item children, then clicking on the menu button and pressing down until you reach the last child in the count. Then press down again to see if it went back to the top.